src/components/NcRichText/autolink.js

```diff
@@ -115,11 +115,26 @@ export const getRoute = (router, url) => {
 
 	const isAbsoluteURL = /^https?:\/\//.test(url)
 
+	// URL is a "mailto:", "tel:" or any custom app protocol link
+	const isNonHttpLink = /^[a-z][a-z0-9+.-]*:.+/.test(url)
```

Comment:
Numbers are valid?

Reply:
According to the standard - yes, except the first character